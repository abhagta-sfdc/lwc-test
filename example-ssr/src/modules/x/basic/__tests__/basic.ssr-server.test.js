import Basic from '../basic';
import { renderAndHashComponent } from '@lwc/jest-ssr-snapshot-utils';

describe('<x-basic>', () => {
    test('should render on the server', async () => {
        const { renderedComponent, snapshotHash } = renderAndHashComponent('x-basic', Basic, {
            msg: 'Welcome!',
        });
        expect(renderedComponent).toMatchSnapshot(snapshotHash);
    });
});
