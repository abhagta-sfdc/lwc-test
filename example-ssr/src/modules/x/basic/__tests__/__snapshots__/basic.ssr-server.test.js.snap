// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<x-basic> should render on the server: 4da3f8ca30774dda15f0aa526a0101c3229e8df7b695c739f96bcf1bc52d3ddd 1`] = `"<x-basic><template shadowrootmode="open"><h1>Basic, Welcome!</h1></template></x-basic>"`;
